### PR TITLE
Project Execution - J1

### PR DESCRIPTION
**Pull request by McDoxa**

### CODE REVIEW

File: project-javascript-j1-objects/Oluwaseyi.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        //6. A function that accepts an object and returns the property value
+        function getPropValue(objectName,property){
+                return objectName[property];
+                };
+        //7. Logging to console.
+        console.log(getPropValue(employee,'name'));
+        console.log(getPropValue(employee,'Phone Number'));
+
+        //8. Adding a method to an object
+        employee.showVisitCard = () => {
```
Comment (basir) [inline project-javascript-j1-objects/Oluwaseyi.html]: showVisitCard should show all info of employee. It only show the property names not values. Change it like this:
name: Doxa
age: 37
...

File: project-javascript-j1-objects/Oluwaseyi.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        //9. Adding another property 'educations' to the object 'employee'.
+        employee.educations = {
+          universtiy1: { gpa: 3.5, year: 2019 },
+          universtiy2: { gpa: 4.0, year: 2015 },
+          universtiy3: { gpa: 3.75, year: 2010 },
+          universtiy4: { gpa: 3.0, year: 2005 }
+        };
+
+        //10. Logging education into console
+        for (let pty in employee.educations){
```
Comment (basir) [inline project-javascript-j1-objects/Oluwaseyi.html]: Change it like an English sentence. `Doxa graduated from University1 in 2019 with GPA 3.5`

File: project-javascript-j1-objects/Oluwaseyi.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+             console.log(`${pty}: {gpa: ${employee['educations'][pty].gpa}, year: ${employee['educations'][pty].year}}`);
+            };
+
+        //11. Calculate the average of the gpa and logging it to console.
+        let totalSum = 0;
+        let number = 0;
+        for (let pty in employee.educations){
```
Comment (basir) [inline project-javascript-j1-objects/Oluwaseyi.html]: It is ok. But you could do it as functional programing:
```
      const gpas = Object.keys(employee.educations).map(
        key => employee.educations[key].gpa
      );
      console.log(gpas.reduce((a, c) => a + c) / gpas.length);
```
It is much easier to read and reasoning.

**Comment by McDoxa** [comment]
Reviewed changes effected. Thank you.

**Pull request merged**